src/josegonzalez/Queuesadilla/Worker/SequentialWorker.php

```diff
@@ -2,11 +2,28 @@
 
 namespace josegonzalez\Queuesadilla\Worker;
 
+declare(ticks=1);
```

Comment:
Yes. It's necessary. When this line is not there, <kbd>Ctrl</kbd>+<kbd>C</kbd> has no effect.